src/ape/managers/project.py

```diff
@@ -2609,6 +2624,29 @@ def chdir(self, path: Optional[Path] = None):
         self.manifest_path = self._base_path / ".build" / "__local__.json"
         self._manifest = self.load_manifest()
 
+    @contextmanager
+    def in_new_path(self, path: Optional[Path] = None):
```

Comment:
got rid of this and made it better but moved it also to a new PR:

https://github.com/ApeWorX/ape/pull/2516